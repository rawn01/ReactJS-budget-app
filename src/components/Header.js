import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => {
   return (
      <header>
         <h1>Budget App</h1>
         <NavLink to="/" activeClassName="is-active" exact={true}>Go home</NavLink><br />
         <NavLink to="/help" activeClassName="is-active">Go to HELP page</NavLink><br />
         <NavLink to="/create" activeClassName="is-active">Go to CREATE page</NavLink>
      </header>
   );
};

export default Header;