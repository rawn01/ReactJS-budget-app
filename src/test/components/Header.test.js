import React from 'react';
import { shallow } from 'enzyme';

import Header from '../../components/Header';


test('should render header correctly', () => {
   const wrapper = shallow(<Header />);
   expect(wrapper).toMatchSnapshot();
});















// import ReactShallowRenderer from 'react-test-renderer/shallow'
// test('should render header correctly', () => {
//    const renderer = new ReactShallowRenderer();
//    renderer.render(<Header />);
//    expect(renderer.getRenderOutput()).toMatchSnapshot();
// });