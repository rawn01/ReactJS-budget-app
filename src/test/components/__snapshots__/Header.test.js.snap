// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Budget App
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
    Go home
  </NavLink>
  <br />
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/help"
  >
    Go to HELP page
  </NavLink>
  <br />
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/create"
  >
    Go to CREATE page
  </NavLink>
</header>
`;
